README.md

```diff
@@ -28,94 +28,17 @@ Decidim is a participatory democracy framework written on Ruby on Rails original
 
 ## What do you need to do?
 
+- [Get started with Decidim](#getting-started-with-decidim)
```

Comment:
I followed the style of the rest of the list, which are answers to the question "What do you need to do?". "Getting started with decidim" would not answer this question (as in it wouldn't be a gramatically correct answer)